packages/SwingSet/src/types-external.js

```diff
@@ -242,39 +242,28 @@ export {};
  */
 
 /**
- * @typedef {object} SwingSetConfig a swingset config object
+ * @typedef {object} SwingSetOptions
  * @property {string} [bootstrap]
  * @property {ConfigProposal[]} [coreProposals]
  * @property {boolean} [includeDevDependencies] indicates that
  * `devDependencies` of the surrounding `package.json` should be accessible to
  * bundles.
- * @property {ManagerType} [defaultManagerType]
  * @property {string} [bundleCachePath] if present, SwingSet will use a bundle cache at this path
- * @property {number} [snapshotInitial]
- * @property {boolean} [pinBootstrapRoot]
- * @property {number} [snapshotInterval]
- * @property {boolean} [relaxDurabilityRules]
  * @property {SwingSetConfigDescriptor} vats
  * @property {SwingSetConfigDescriptor} [bundles]
  * @property {BundleFormat} [bundleFormat] the bundle source / import bundle
  * format.
  * @property {*} [devices]
  */
+/**
+ * @typedef {KernelOptions & SwingSetOptions} SwingSetConfig a swingset config object
+ */
 
 /**
- * @typedef {object} SwingSetKernelConfig the config object passed to initializeKernel
- * @property {string} [bootstrap]
- * @property {boolean} [includeDevDependencies] indicates that
- * `devDependencies` of the surrounding `package.json` should be accessible to
- * bundles.
```

Comment:
Are we losing this bit of documentation? was it not true?

Reply:
this was a dupe. still in the file: https://github.com/Agoric/agoric-sdk/blob/fc700595eb1cd9114b5613e0d267797c0bc948e0/packages/SwingSet/src/types-external.js#L248-L251